fcrepo-clients/src/test/java/edu/unc/lib/dl/acl/fcrepo4/AccessControlServiceImplIT.java

```diff
@@ -51,9 +50,6 @@
  */
 public class AccessControlServiceImplIT {
```

Comment:
This test is failing for me when i run all tests, but not when i run it by itself.  NPE in edu.unc.lib.dl.fedora.ContentPathFactory at line 74.  Doesn't fail when i run it individually though